Backend-API/src/main/java/com/avangers/backendapi/services/UserService.java

```diff
@@ -11,4 +13,5 @@ public interface UserService extends UserDetailsService {
   RegisterUserResponseDTO addUser(RegisterUserRequestDTO registerUserRequestDTO);
   DeleteUserResponseDTO deleteUser(String email);
   UpdateUserResponseDTO updateUser(String email, UpdateUserRequestDTO updateUserRequestDTO);
+  LoginResponseDTO login(LoginRequestDTO loginRequestDTO);
```

Comment:
This method does not follow our convention. (Every other method has the `User` suffix.) It is debatable if we need those or not, but we should follow some conventions.